docs/datatypes/has.md

```diff
@@ -0,0 +1,6 @@
+---
+id: datatypes_has
+title:  "Has"
+---
+The trait `Has[A]` is used with ZIO environment to express an effect'sdependency on a service of type `A`. <br>
```

Comment:
`effect'sdependency` =>`effect's dependency`

Reply:
Yes, thank you! Will update now